modules/logging-loki-zone-fail-recovery.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-loki.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="logging-loki-zone-fail-recovery_{context}"]
+= Recovering Loki pods from failed zones
+
+In {product-title} a zone failure happens when specific availability zone resources become inaccessible. Availability zones are isolated areas within a cloud provider's data center, aimed at enhancing redundancy and fault tolerance. If your {product-title} cluster isn't configured to handle this, a zone failure can lead to service or data loss.
+
+Loki pods are part of a link:https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/[StatefulSet], and they come with Persistent Volume Claims (PVCs) provisioned by a `StorageClass`. Each Loki pod and its PVCs reside in the same zone. When a zone failure occurs in a cluster, the StatefulSet controller will automatically attempt to recover the affected pods in the failed zone.
+
+[WARNING]
+====
+The following procedure will delete the PVCs in the failed zone, and all data contained therein.  To avoid complete data loss the replication factor field of the `LokiStack` CR should always be set to a value greater than 1 to ensure that Loki is replicating.
+====
+
+.Prerequisites
+* Logging version 5.8 or later.
+* Verify your `LokiStack` CR has a replication factor greater than 1.
+* Zone failure detected by the control plane, and nodes in the failed zone are marked by cloud provider integration.
+
+While the StatefulSet controller will automatically attempt to reschedule pods in a failed zone, since the associated PVCs are also in the failed zone, automatic rescheduling to a different zone will not work. PVCs in the failed zone must be manually deleted to allow successful recreation of the stateful Loki Pod and its provisioned PVC in the new zone.
```

Comment:
Per ISG, use _while_ and _since_ only referring to a period of time.  Also, per ISG do not use _recreate_.

```suggestion
The StatefulSet controller automatically attempts to reschedule pods in a failed zone. Because the associated PVCs are also in the failed zone, automatic rescheduling to a different zone does not work. You must manually delete the PVCs in the failed zone to allow successful re-creation of the stateful Loki Pod and its provisioned PVC in the new zone.
```